### PR TITLE
Changes from Atalay.

### PR DESCRIPTION
**Pull request by xsub**
This is the code from @sakelestemur's `almalinux-responsive` archive.
Just fixed a couple of trailing spaces, empty lines, etc. 

Side note:
We probably need to clean up the html as `tidy` reports a bunch of errors.

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -310,7 +353,7 @@ <h4 class="fw-bold mb-0 pb-2">{{ i18n "Fully supported, always free" }}</h4>
                         <div>
                             <h4 class="fw-bold mb-0 pb-2">{{ i18n "Effortless to switch to" }}</h4>
                             <p>
-                                {{ i18n "Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS."}} {{ i18n "Switching is easy" }} - <a href="https://github.com/AlmaLinux/almalinux-deploy">{{ i18n "read how" }}.</a>
+                                Switching distributions is usually costly and time-consuming, but that’s not the case when switching from CentOS and other RHEL® derivatives to AlmaLinux OS. Switching is easy  - <a href="https://github.com/AlmaLinux/almalinux-deploy">{{ i18n "read how" }}</a>.
```
Comment (codyro) [inline layouts/index.html:356]: Any reason localization was removed from this paragraph? If not please re-add it before I merge it in (or if you like I can).

**Comment by bennyvasquez** [comment]
@xsub I agree with @codyro. The changes seem sane, excepting the removal of the localization integration. Update that, and we should be good to merge, and then we can start testing further.

File: layouts/index.html (hunk 1 above)
Comment (xsub) [reply layouts/index.html:356]: I have no idea, we need to ask @sakelestemur if that was intentional.  I did check and this how it looks in the archive he provided me with.  I think that this was out of scope of the changes he was doing.

Comment (codyro) [reply layouts/index.html:356]: I'm guessing he didn't pull in the latest changes so it was just missed. You can go ahead and re-wrap that string with the `{{ i18n " " }}` tags and then we can merge everything.

Comment (xsub) [reply layouts/index.html]: Done.

**Comment by codyro** [comment]
I forgot to address your `tidy` comment. We will beautify & tidy the code prior to production however the plan will be to add it to the CI pipeline to automate it down the road.

**Pull request merged**